### ISSUE TITLE
instantiate CSGSphere3D or CSGBox3D -> crash

### ISSUE THREAD

**Issue by noideaforaccountname** [godot-nim/gdext-nim#132]
if i do like written in wiki with meshinstance3d it works:
`  let m : MeshInstance3D = instantiate MeshInstance3D
  if m == nil:
    print "node not found"
  else:
    print "csgsphere: ", m`
output: csgsphere: <MeshInstance3D#27917288714>

but if i do the same with one of the CSGs like CSGSphere3D the program is compiled without error and 
the program crashes  (without any error). whats wrong please?

**Comment by panno8M** [comment]
This seems to be a bug on the library side. Please wait a little while as we will fix it later today.

**Comment by panno8M** [comment]
Fixed, please reinstall 0.3.1.

**Comment by panno8M** [comment]
#### Small tips for Markdown

If you want to attach an Nim code, try writing:

<span>
```nim<br>
let m : MeshInstance3D = instantiate MeshInstance3D<br>
if m == nil:<br>
&nbsp;&nbsp;print "node not found"<br>
else:<br>
&nbsp;&nbsp;print "csgsphere: ", m<br>
```
</span>

The above example would appear as follows:

```nim
let m : MeshInstance3D = instantiate MeshInstance3D
  if m == nil:
    print "node not found"
  else:
    print "csgsphere: ", m
```
